src/core/lib/transport/transport.cc

```diff
@@ -36,9 +36,6 @@
 #include "src/core/lib/slice/slice_string_helpers.h"
 #include "src/core/lib/transport/transport_impl.h"
 
-grpc_core::DebugOnlyTraceFlag grpc_trace_stream_refcount(false,
```

Comment:
We still need the definition here.

But we can remove the declaration in the .h file.

Reply:
This is my bad. But seems like this declaration should remain in the .h file because some inline functions are referring it.